src/js/chrome/entry.js

```diff
@@ -57,7 +58,11 @@ export function chromeInit(navResolver) {
 
   return {
     identifyApp: (data) => navResolver.then(() => identifyApp(data, store.getState().chrome.globalNav)),
-    navigation: () => console.error("Don't use insights.chrome.navigation, it has been deprecated!"),
+    navigation: () => {
+      log('Deprecated function call: insights.chrome.navigation');
+      Sentry.captureMessage('Deprecated function call: insights.chrome.navigation');
+      console.error("Don't use insights.chrome.navigation, it has been deprecated!");
```

Comment:
You don't have to do two of these. If you just throw a `console.error` it'll send to sentry automatically

Reply:
Ah so just `console.error("Don't use insights.chrome.navigation, it has been deprecated!");` is enough? Ok, so I guess no changes are required for this part. It was like that before.